ios/TransparentVideoViewManager.swift

```diff
@@ -46,7 +46,7 @@ class TransparentVideoView : UIView {
           (kCVPixelBufferPixelFormatTypeKey as String): kCVPixelFormatType_32BGRA]
 
       // Setup looping on our video
-      playerView.isLoopingEnabled = true
+      playerView.isLoopingEnabled = false
```

Comment:
Hey @ibrkhalil, thanks for contributing to this repository!

This change would break other use cases where we need videos to loop, for example the parallax effect for Enable Biometrics screen in `status-mobile`. So, the best approach would be to create a new `loop` property in the component and set it here, and pass the appropriate prop value when rendering the component.

Also we would need to implement this change on Android. I know https://github.com/status-im/status-mobile/issues/16827 only mentions the issue on iOS, but this is because we are currently rendering images on `status-mobile` on Android. When making change to this library, we should make sure native changes are reflected on both platforms.

Reply:
Okay cool, investigating